llvm/lib/Target/AMDGPU/Utils/AMDGPUWaitCountUtils.cpp

```diff
@@ -0,0 +1,1393 @@
+//===-- AMDGPUWaitCountUtils.cpp ------------------------------------------===//
```

Comment:
I think the `Utils` directory is only for things shared between codegen and asm/dis. This should go in the main `AMDGPU` directory.